runbotics/runbotics-orchestrator-ui/src/providers/BpmnForm.provider.tsx

```diff
@@ -35,9 +37,18 @@ const BpmnFormProvider: FC<BpmnFormProviderProps> = ({ modeler, element, childre
     }, [process]);
 
     return (
-        <BpmnFormContext.Provider value={{ modeler, element, action, setAction, passedInVariables }}>
-            {children}
-        </BpmnFormContext.Provider>
+        <If
```

Comment:
imo this is not the best place for jsx message, move it to the appropriate component